### ISSUE TITLE
Kinetics pretrained weights

### ISSUE THREAD

**Issue by jinwchoi** [yjxiong/action-detection#90]
Hi @yjxiong, 

I found Kinetics pre-trained BNInception RGB weight here:
https://yjxiong.blob.core.windows.net/ssn-models/bninception_rgb_kinetics_init-d4ee618d3399.pth
I found this from SSN codebase(https://github.com/yjxiong/action-detection).

Since SSN and TSN share the same backbone (BNInception or Inception V3), I guess I can use this weight for finetuning TSN on other datasets such as UCF101 or HMDB51. Is it verified that you can reproduce UCF101/HMDB51 results finetune from Kinetics pretrained PyTorch weights? 

I am pointing out this since on your "TSN Pretrained Models on Kinetics Dataset" webpage, there are only Caffe weights but no PyTorch weights pretrained on Kinetics. How about explicitly show PyTorch weights as well? 

Thanks!

**Comment by yjxiong** [comment]
I’m not sure this will reproduce the results but you can try. The best way is to use the Caffe TSN and the pretrained weights from the websites.

**Comment by jinwchoi** [comment]
Thanks for your response. I just wanted to use PyTorch instead of Caffe. Let me try this out.

Are there any specific details such as learning rate schedule, when you finetune your Kinetices pretrained model on UCF101/HMDB51?

**Comment by yjxiong** [comment]
Not much specifics. Just fix all bn layers.

**Comment by jinwchoi** [comment]
So when finetune from ImageNet pretrained model, you fix all BN but first one,
Finetuning from ImageNet+Kinetics pretrained model, you fix literally all the BN layers? Is it correct?

**Comment by yjxiong** [comment]
Yes. 

Also we will not have weights trained in PyTorch for this setting to share online. Because we don’t want to encourage more of this kind of experiments. UCF101 is almost saturated nowadays. Any new work should be validated on larger, more challenging datasets, e.g. Kinetics, Something Something, etc.
